keps/sig-scheduling/5004-dra-extended-resource/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA Extended Resource
+kep-number: 5004
+authors:
+  - "@yliaog"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+status: draft
```

Comment:
This will need to be set to `implementable` to be included in the release, and we will need a PRR file (with me as approver) as well.

Reply:
done